### DISCUSSION TITLE
Any chance there would be an Android port??

### DISCUSSION THREAD

**Discussion by yumnycheesecakes** [stepmania/stepmania#2248] [Q&A]
Hi everyone, I like this game very much (knew it long, started playing in 2019) and I have a mobile phone on Android. I wanna play this game on it. There would be a port?? I played unofficial versions, but it lacks many features.

**Comment by Horsey-** [comment]
IIRC Android has really low standards for audio latency. There can be 20ms or more added just by playing on the Android OS.

**Comment by geefr** [comment]
It's technically possible, I compiled parts of sm ages ago as a test.
SMX is I think Android underneath? Obviously they're not sharing their code, but proves it's possible, there's also been a few android apps based on SM (I assume 3.95 but it's similar).

The latency thing is definitely an issue, my phone gets around 100ms in rhythm games.

Finding someone willing to put the effort in may be a challenge - I might be eventually for a silly idea, but at most it'd be a prototype / curiosity build.

**Comment by 88572** [comment]
BeatX can play StepMania songs!
https://play.google.com/store/apps/details?id=sk.maniacs.bm

**Reply by yumnycheesecakes** [reply]
i tried beatx, but it lacks some features that stepmania has
